docs/demos/onvif-authentication-access.md

```diff
@@ -0,0 +1,306 @@
+
+```bash=
+$ kubectl get nodes,akric,akrii,pods
+NAME           STATUS   ROLES           AGE   VERSION
+node/kube-01   Ready    control-plane   22d   v1.26.1
+node/kube-02   Ready    <none>          22d   v1.26.1
+node/kube-03   Ready    <none>          22d   v1.26.1
+
+NAME                               CAPACITY   AGE
+configuration.akri.sh/akri-onvif   3          62s
+
+NAME                                 CONFIG       SHARED   NODES                   AGE
+instance.akri.sh/akri-onvif-029957   akri-onvif   true     ["kube-03","kube-02"]   48s
+
+NAME                                              READY   STATUS    RESTARTS   AGE
+pod/akri-agent-daemonset-gnwb5                    1/1     Running   0          62s
+pod/akri-agent-daemonset-zn2gb                    1/1     Running   0          62s
+pod/akri-controller-deployment-56b9796c5-wqdwr    1/1     Running   0          62s
+pod/akri-onvif-discovery-daemonset-wcp2f          1/1     Running   0          62s
+pod/akri-onvif-discovery-daemonset-xml6t          1/1     Running   0          62s
+pod/akri-webhook-configuration-75d9b95fbc-wqhgw   1/1     Running   0          62s
+pod/kube-02-akri-onvif-029957-pod                 1/1     Running   0          48s
+pod/kube-03-akri-onvif-029957-pod                 1/1     Running   0          48s
+```
+Dump the environment variables from the container to check the device uuid from the container's environment variables. Below is an example, the Onvif discovery handler discovers the camera and expose the device's uuid. **Write down the device uuid for later use**. Note that in real product scenarios, the device uuids are acquired directly from the vendors or already known before installing Akri Configuration.
+
+```bash=
+$ kubectl exec -i kube-02-akri-onvif-029957-pod -- /bin/sh -c "printenv|grep ONVIF_DEVICE"
+ONVIF_DEVICE_SERVICE_URL_029957=http://192.168.1.145:2020/onvif/device_service
+ONVIF_DEVICE_UUID_029957=3fa1fe68-b915-4053-a3e1-ac15a21f5f91
+```
+
+#### Set up Kubernetes secrets
+Now we can set up the credential information to Kubernetes Secret. Replace the device uuid and the values of username/password with information of your camera.
+
+```bash
+cat > /tmp/onvif-auth-secret.yaml<< EOF
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: onvif-auth-secret
+type: Opaque
+stringData:
+  device_credential_list: |+
+    [ "credential_list" ]
+  credential_list: |+
+    {
+        "3fa1fe68-b915-4053-a3e1-ac15a21f5f91" :
+            {
+                "username" : "camuser",
+                "password" : "HappyDay"
+            }
+    }
+EOF
+
+# add the secret to cluster
+kubectl apply -f /tmp/onvif-auth-secret.yaml
+
+```
+#### Upgrade the Akri configuration
+Upgrade the Akri Configuration to include the secret information and the sample video broker container.
+
+```bash
+helm upgrade akri akri-helm-charts/akri-dev \
```

Comment:
add the `--install` flag in case the user skipped previous step.

Reply:
add --install